_data/security.yml

```diff
@@ -70,6 +70,12 @@ websites:
       - totp
     doc: https://cobalt.io/blog/new-feature-two-factor-auth/
 
+  - name: Crowdstrike
+    url: https://falcon.crowdstrike.com
```

Comment:
```suggestion
    url: https://crowdstrike.com
```

Reply:
Well we need some way of verifying it so either upload a screenshot of the documentation or the 2fa process 🤷 
The `url` tag is supposed to point to the homepage of the service and it can't be a login page. `https://crowdstrike.com` seems to meet both those requirements.

Reply:
There seems to be some documentation for Crowdstrike 2FA from Dell Support as well: https://www.dell.com/support/kbdoc/en-uk/000126191